src/features/unified-share-modal/SharedLinkAccessDescription.js

```diff
@@ -3,23 +3,31 @@ import React from 'react';
 import { FormattedMessage } from 'react-intl';
 
 import type { ItemType } from '../../common/types/core';
+import { isBoxCanvas } from '../../utils/file';
+import convertToBoxItem from './utils/item';
 
 import { ANYONE_WITH_LINK, ANYONE_IN_COMPANY, PEOPLE_IN_ITEM } from './constants';
-import type { accessLevelType } from './flowTypes';
+import type { accessLevelType, item as itemtype } from './flowTypes';
```

Comment:
were you unable to just import `item`as is, there are other types that do not have the word "Type" at the end and they are imported as is?

Reply:
The thing is that there's either an argument or a constant called `item` that is coming from `Props`, so there could potentially be a conflict with those 🤔 Also, I followed what was already done in SharedLinkSection.js. Should we try and just use `item` as is here?

Reply:
yeah looks to be done that way for naming conflicts. seems okay to keep